src/components/views/rooms/MemberList.js

```diff
@@ -32,7 +32,7 @@ module.exports = React.createClass({
     displayName: 'MemberList',
 
     getInitialState: function() {
-        this.memberDict = this.getMemberDict();
+        this._mounted = false;
```

Comment:
I'm not sure why this was side-effecting in `getInitialState` - it should probably in `componentWillMount`.

Reply:
Maybe I misunderstand, but I removed this line because there was no reason to cache the `memberDict` as it was only being used by `roomMembers()`, and as I convert `room.currentState.members` to an array as early as possible I did away with the `memberdict` and call it straight from `roomMembers()`

Reply:
ah, sorry, I somehow read that your comment was related to the line above. will do :+1: